public/locales/kn.json

```diff
@@ -1,21 +1,21 @@
 {
-  "We stand with everyone fighting on the frontlines": "ಮುಂಚೂಣಿಯಲ್ಲಿ ಹೋರಾಡುವ ಪ್ರತಿಯೊಬ್ಬರೊಂದಿಗೆ ನಾವು ನಿಲ್ಲುತ್ತೇವೆ",
-  "Cumulative": "ಸಂಚಿತ",
-  "Spread Trends": "ಟ್ರೆಂಡ್‌ಗಳನ್ನು ಹರಡಿ",
+  "We stand with everyone fighting on the frontlines": "ಮುಂದೆ ನಿಂತು ಹೋರಾಡುತ್ತಿರುವ ಪ್ರತಿಯೊಬ್ಬರೊಂದಿಗೆ ನಾವಿದ್ದೇವೆ",
+  "Cumulative": "ಒಟ್ಟು",
+  "Spread Trends": "ಹರಡುವಿಕೆ ದಿಕ್ಕು",
   "from": "ನಿಂದ",
-  "India COVID-19 Tracker": "ಭಾರತ COVID-19 ಟ್ರ್ಯಾಕರ್",
-  "A Crowdsourced Initiative": "ಎ ಕ್ರೌಡ್‌ಸೋರ್ಸ್ಡ್ ಇನಿಶಿಯೇಟಿವ್",
+  "India COVID-19 Tracker": "ಭಾರತ COVID-19 ಜಾಡುಗಾರ",
+  "A Crowdsourced Initiative": "ಸಾರ್ವಜನಿಕರ ಪಾಲುಗಾರಿಕೆಯಲ್ಲಿ",
   "Ago": "ಹಿಂದೆ",
-  "Scale Uniformly": "ಏಕರೂಪವಾಗಿ ಅಳೆಯಿರಿ",
-  "Checked by default to scale uniformly": "ಏಕರೂಪವಾಗಿ ಅಳೆಯಲು ಪೂರ್ವನಿಯೋಜಿತವಾಗಿ ಪರಿಶೀಲಿಸಲಾಗಿದೆ.",
+  "Scale Uniformly": "ಸಮ ಅಂತರದಲ್ಲಿ ಬಿಂಬಿಸು",
+  "Checked by default to scale uniformly": "ಸಮ ಅಂತರದಲ್ಲಿ ಬಿಂಬಿಸಲು ಪೂರ್ವನಿಯೋಜಿತವಾಗಿ ಆಯ್ಕೆ ಮಾಡಿದೆ",
   "Last Updated": "ಕೊನೆಯದಾಗಿ ನವೀಕರಿಸಲಾಗಿದೆ",
-  "Daily": "ದೈನಂದಿನ",
-  "Confirmed": "ದೃ .ಪಡಿಸಲಾಗಿದೆ",
+  "Daily": "ದೈನಿಕ",
+  "Confirmed": "ಖಚಿತ",
   "Active": "ಸಕ್ರಿಯ",
-  "Recovered": "ಚೇತರಿಸಿಕೊಂಡ",
-  "Deceased": "ಮರಣ",
+  "Recovered": "ವಾಸಿ",
```

Comment:
Prefer ಗುಣವಾದವರು here. Vaasi is mostly south Karanataka specific word.

Reply:
Had used ವಾಸಿ since it was short. Yes, will use ಗುಣವಾದವರು in my next PR

Reply:
Checked with couple of Kannada friends; they suggested guNamukha - this should be shorter I hope.

Reply:
Sure. Point noted

Reply:
Changed to ಗುಣಮುಖ now